app/views/api/v2/hosts/show.json.rabl

```diff
@@ -25,3 +25,16 @@ end
 child :config_groups do
   extends "api/v2/config_groups/main"
 end
+
+host_additional_views(@host).each do |id, tab|
+  if tab.is_a? String
+    node do |host|
+      partial "api/v2/#{tab}", :object => host
+    end
+  else
+    child tab => id.to_sym do |facet|
+      class_name = tab.class.name.demodulize.underscore
+      extends "api/v2/#{class_name.pluralize}/base"
+    end
+  end
+end
```

Comment:
Any thoughts on adding support for base.json.rabl as well so we can modify the index results?